packages/database/src/entities/index.ts

```diff
@@ -1 +1,9 @@
+/* istanbul ignore file */
 export * from './User';
+export * from './JudgingVote'
+export * from './Project'
+export * from './AppConfig'
+export * from './Event'
+export * from './Judge'
+export * from './Node'
```

Comment:
Node is the one exception; the API will never need to interact with that so it should be omitted here
```suggestion
```